### PR TITLE
Add meta example for laplacian eigen maps converter.

### PR DESCRIPTION
**Pull request by Hephaestus12**

### CODE REVIEW

**Comment by karlnapf** [comment]
ok

File: examples/meta/src/converter/converter_laplacian_eigen_maps.sg.in [hunk 1]
```diff
@@ -0,0 +1,17 @@
+File f_feats_train = read_csv("@SHOGUN_DATA@/fm_train_real.dat")
+
+#![create_features]
+Features feats = create_features(f_feats_train)
+#![create_features]
+
+#![set_parameters]
+Transformer fa = create_transformer("LaplacianEigenmaps", target_dim=1, k=20, tau=20)
```
Comment (karlnapf) [inline examples/meta/src/converter/converter_laplacian_eigen_maps.sg.in]: tau needs to be a float.
Please test those things before you send a PR

```
make build_cpp_meta_examples
ctest -R laplacian_eigen_maps
```

File: examples/meta/src/converter/converter_laplacian_eigen_maps.sg.in [hunk 2]
```diff
@@ -0,0 +1,17 @@
+File f_feats_train = read_csv("@SHOGUN_DATA@/fm_train_real.dat")
```
Comment (karlnapf) [inline examples/meta/src/converter/converter_laplacian_eigen_maps.sg.in]: could you also pls remove "converter_" from the filename of all of those converter examples. The folder has that information in them

File: examples/meta/src/converter/converter_laplacian_eigen_maps.sg.in (hunk 1 above)
Comment (karlnapf) [reply examples/meta/src/converter/converter_laplacian_eigen_maps.sg.in]: Make sure to read the error msgs from the CI. And also *please test meta examples locally before you send them*. This doesnt execute.

https://dev.azure.com/shogunml/shogun/_build/results?buildId=3927&view=logs&j=014b6934-7273-5d37-4c53-a79dadc49884&t=53126599-58a0-5542-c2f0-17b2aa146982&l=345

File: examples/meta/src/converter/converter_laplacian_eigen_maps.sg.in (hunk 2 above)
Comment (Hephaestus12) [reply examples/meta/src/converter/converter_laplacian_eigen_maps.sg.in]: Yes, I'll do this.

File: examples/meta/src/converter/converter_laplacian_eigen_maps.sg.in (hunk 1 above)
Comment (Hephaestus12) [reply examples/meta/src/converter/converter_laplacian_eigen_maps.sg.in]: Yes, sorry. I just tested one of them and then pushed all. I'll be more careful.

Comment (karlnapf) [reply examples/meta/src/converter/converter_laplacian_eigen_maps.sg.in]: no worries, I think it should be very little work to test all compared to one.

**Pull request closed without merging**